### ISSUE TITLE
Can't import node modules like angular2 in 0.19.1

### ISSUE THREAD

**Issue by johnpapa** [systemjs/systemjs#805]
In 0.19.0 and 0.18.x I was able to do the following, but now I get errors in 0.19.1 when it tries to import any angular2 files.

```
    System.config({
      packages: {
        'app': {defaultExtension: 'js'}
      }
    });
    System.import('app/bootstrap').catch(console.log.bind(console));
```

**Comment by johnpapa** [comment]
More info

![image](https://cloud.githubusercontent.com/assets/1202528/10038297/ffd22b78-6191-11e5-8597-188724edd2af.png)

**Comment by johnpapa** [comment]
I also tried map, path, and a variety of other changes. No luck

cc @wardbell and @simpulton

**Comment by guybedford** [comment]
@johnpapa is there a repo I can clone to look into this? It sounds like you want to set `defaultJSExtensions: true` or enable `angular2` itself as a package.

**Comment by guybedford** [comment]
Ok yes this is a bug, I'll aim to post out a patch now.

**Comment by guybedford** [comment]
Fixed in https://github.com/systemjs/systemjs/commit/0a5dda604e9008b6aa5e10ac0ce39fff72d6e5f4.

**Comment by guybedford** [comment]
Released in 0.19.2.

**Comment by johnpapa** [comment]
Thanks for the fast response. It works again
